frontend/src/pages/HomePage/index.js

```diff
@@ -5,19 +5,21 @@ import Title from "components/Typography/Title";
 
 import ellipse1 from "assets/imgs/ellipses/Ellipse1.svg";
 import styles from "./index.module.css";
+import { COMPANY, WHEEL_CONFIG } from "constants/Subvisual.js";
 
 const HomePage = () => {
   return (
     <div>
-      <Header>Subvisual</Header>
+      <Header>{COMPANY}</Header>
       <div>
```

Comment:
Is this `div` necessary?

Reply:
It was only there so react didn't complain, so I can switch that for <></>